src/test/fuzz/util.h

```diff
@@ -340,4 +342,11 @@ void ReadFromStream(FuzzedDataProvider& fuzzed_data_provider, Stream& stream) no
     }
 }
 
+inline void FinalizeHeader(CBlockHeader& header, const ChainstateManager& chainman)
```

Comment:
nit: please mention in the commit message why the method was slightly changed besides the move